supplementary_style_guide/introduction/about-guide.adoc

```diff
@@ -1,21 +1,28 @@
 = About this guide
 
-This guide provides style guidelines and preferred term usage for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. It is intended to be used as a supplement to the _IBM Style_ guide, which is the primary source of guidance for Red Hat product documentation.
+The style and language guidance in this guide overrides or supplements some guidance provided by _IBM Style_, which is the primary source of style guidance for link:https://www.redhat.com/[Red Hat] product and cross-product solution documentation. 
 
-The Red Hat Customer Content Services team created this guide as part of its responsibility for producing product documentation for Red Hat customers. Upstream communities who want to align more closely with the standards used by Red Hat product documentation can also use this guide.
+The Red{nbsp}Hat Customer Content Services team has created this guide as part of its responsibility for producing product documentation for Red{nbsp}Hat customers. Upstream communities who want to align more closely with the standards used by Red{nbsp}Hat product documentation can also use this guide.
 
 [NOTE]
 ====
-Other Red Hat technical documentation, including Red Hat training and exam content by Global Learning Services (GLS), follows the link:https://stylepedia.net/[_Red Hat Technical Writing Style Guide_] instead of the _Red Hat supplementary style guide for product documentation_.
+Other Red Hat technical documentation, including Red Hat training and exam content by Global Learning Services (GLS), follows the link:https://stylepedia.net/[_Red Hat Technical Writing Style Guide_] instead of the _Red{nbsp}Hat supplementary style guide for product documentation_.
 ====
 
-== How to use this guide
+== Using style guides for Red{nbsp}Hat product documentation
 
-When looking for style guidance, reference this guide first, because it overrides certain guidance from the _IBM Style_ guide. If you must deviate from the guidance in either guide, notify the style council by opening an link:https://github.com/redhat-documentation/doc-style/issues[issue]. This way, the deviation can be discussed by the style council and included in this guide.
+Red{nbsp}Hat product documentation adheres to the style guidance provided in _IBM Style_, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as Red{nbsp}Hat Enterprise Linux or Red{nbsp}Hat Openshift Container Platform.
```

Comment:
Same about the ISG wording. And also I think just one example of a product-specific guide is sufficient. Esp since we don't talk about the RHEL one again, but we do the OCP one in the graphic.

Also, do you think we should link to the OCP one since it's public? I don't feel strongly about it so I didn't include it in my suggestion below, just just something to consider.

```suggestion
Red{nbsp}Hat product documentation adheres to the style guidance provided in the _IBM Style_ guide, the _Red{nbsp}Hat supplementary style guide for product documentation_, and style guides specific to Red{nbsp}Hat products, such as the Red{nbsp}Hat Openshift documentation guidelines.
```

Reply:
I included those two because those are the two biggest sets of RH product docs. I'd prefer to leave a link out because we don't provide links for anything else in this section. So leaving it as-is.